packages/blocks/table/src/App.tsx

```diff
@@ -368,7 +368,7 @@ export const App: BlockComponent<AppProps> = ({
 
   /** @todo Fix keys in iterators below to not use the index */
   return (
-    <div>
+    <div className={tw`overflow-x-auto`}>
```

Comment:
This overflow tag allows the table to overflow well. Adding an `overflow-x` to its parent results in some columns being hidden in mobile view.